README.md

```diff
@@ -0,0 +1,10 @@
+[![Build Status](https://travis-ci.org/openshift/openshift-state-metrics.svg?branch=master)](https://travis-ci.org/wanghaoran1988/openshift-state-metrics)
```

Comment:
@wanghaoran1988 link needs fixing: https://travis-ci.org/wanghaoran1988/openshift-state-metrics

Will this be fixed once we get travis running for the new repo?

Reply:
yes